airflow/utils/platform_utils.py

```diff
@@ -0,0 +1,16 @@
+import platform
+
+def is_windows() -> bool:
+    """
+    Returns true if executing system is Windows
+    """
+    return platform.system() == 'Windows'
```

Comment:
We don't need a function for this either.

```suggestion
IS_WINDOWS = platform.system() == 'Windows'
"""True if operating system is Windows"""```

Reply:
True, has been changed in the latest commit.